docs/content/release_notes/changelog_1.9.0.md

```diff
@@ -2,3 +2,4 @@
 
 IBM® block storage CSI driver 1.9.0 adds new support and enhancements:
 - New stretched topology support for SAN Volume Controller storage systems
+- Additional orchestration platform support for OpenShift 4.10 and Kubernetes 1.23
```

Comment:
```suggestion
- Additional orchestration platforms support for OpenShift 4.10 and Kubernetes 1.23
```

Reply:
i'm not changing this. "platform support" is correct